example/settings.py

```diff
@@ -82,9 +82,15 @@
 
 ES_DISABLED = False
 
-ES_URLS = ['http://localhost:9200']
+ES_URLS = ['http://elasticsearch.local:9200']
```

Comment:
Agree, but I wasn't able to get it forwarded on localhost. Do you know what docker-compose setting to get it forwarded properly?